tools/benchcomp/benchcomp/cmd_args.py

```diff
@@ -138,6 +138,13 @@ def _get_args_dict():
                         "help":
                             "do not make a fresh copy of the benchmark "
                             "directories before running each variant",
+                    }, {
+                        "flags": ["--no-cleanup-run-dirs"],
```

Comment:
You could use a similar name to Kani `--keep-temps` or `--keep-run-dirs`.

Reply:
Done, thank you